site-src/concepts/guidelines.md

```diff
@@ -111,4 +111,13 @@ cases. One variation from the Kubernetes conventions is for fields with
 default values. These fields are considered optional but are not pointers
 since they will never be empty.
 
+Another convention this project uses is for plural field names for lists
```

Comment:
🤦  I'd missed updating the paragraph above which is unfortunately quite outdated. It looks like we need to fix the "Service APIs" reference and the variation described in that paragraph is no longer true. I think it might make sense to restructure all of the above to something like this:

```md
Gateway API follows Kubernetes API [conventions][1]. These conventions
are intended to ease client development and ensure that configuration
mechanisms can consistently be implemented across a diverse set of use
cases. In addition to the Kubernetes API conventions, Gateway API has the
following conventions:

### List Names
...
```

Reply:
Hah, I missed that too. I'll update.